### ISSUE TITLE
Battery bum bug

### ISSUE THREAD

**Issue by Eris-GitHub** [UnderscoreKilburn/repentance-issues#396]
You can't recharge your card slot active item on tСharacters using battery bum even if your first active item is already charged.

**Comment by Zamiell** [comment]
The active item is relevant, but not in the way that OP describes.

For example:
- Start a run on Tainted Magdalene.
- Spawn a Battery Bum.
- `g dollar`
- Use the Battery Bum until it pays out with a charge.
- The Yum Heart pocket item will never be charged. (bug)

For example:
- Start a run on Tainted Magdalene.
- Spawn a Battery Bum.
- `g dollar`
- `g d6`
- Use the Battery Bum until it fully double-charges the D6.
- Continue to use the Battery Bum.
- The Yum Heart pocket item will be charged as expected once the D6 is fully double-charged. (not bugged)

**Comment by eyhung** [comment]
Correct.  I got this with Tainted Lazarus holding no active item in either incarnation.  Battery Bum does not charge the Flip pocket item until an active item is held.
